e2e_test/backfill/backfill_order_control.slt

```diff
@@ -0,0 +1,83 @@
+statement ok
+SET RW_IMPLICIT_FLUSH TO TRUE;
+
+statement ok
+drop table if exists car_sales cascade;
+
+statement ok
+drop table if exists car_info cascade;
+
+statement ok
+drop table if exists car_regions cascade;
+
+statement ok
+create table car_sales(id int, car_id int, region_id int, price int);
+
+statement ok
+create table car_info(id int, name varchar);
+
+statement ok
+create table car_regions(id int, region varchar);
+
+statement ok
+insert into car_info select t.id as id, 'unmatched' from generate_series(10, 100000) t(id);
+
+statement ok
+insert into car_info values (1, 'Toyota'), (2, 'Honda'), (3, 'Ford'), (4, 'Chevy'), (5, 'BMW'), (6, 'Audi'), (7, 'Mercedes');
+
+statement ok
+insert into car_regions select t.id as id, 'unmatched' from generate_series(10, 100000) t(id);
+
+statement ok
+insert into car_regions values (1, 'North America'), (2, 'Europe'), (3, 'Asia'), (4, 'South America'), (5, 'Africa'), (6, 'Australia'), (7, 'Antarctica');
+
+# Generate 100K sales per car per region
+statement ok
+INSERT INTO car_sales
+  SELECT
+    sales_id_gen.id as id,
+    car_id_gen.id as car_id,
+    region_id_gen.id as region_id,
+    sales_id_gen.id as price
+  FROM generate_series(1, 100000) as sales_id_gen(id)
+    CROSS JOIN generate_series(1, 5) as car_id_gen(id)
+    CROSS JOIN generate_series(1, 5) as region_id_gen(id);
+
+# Create an MV that groups sales by a range of prices
+statement ok
+create materialized view m1
+    with (backfill_order = FIXED car_regions -> car_sales, car_info -> car_sales)
```

Comment:
Shall we encapsulate the order value into a string, so that we don't break the Postgres syntax too much? For example, this can be more friendly to formatters.

Reply:
Yeah, I will do something like `FIXED("car_regions -> car_sales, ...")` instead.

Reply:
`FIXED(car_regions -> car_sales, ...)`.

Because the tokenizer will take `'car_regions -> car_sales, ...'` as a single token.